com.unity.render-pipelines.high-definition/Runtime/Material/Lit/Lit.shader

```diff
@@ -241,6 +241,7 @@ Shader "HDRP/Lit"
     HLSLINCLUDE
 
     #pragma target 4.5
+    //#pragma enable_d3d11_debug_symbols
```

Comment:
can i leave that here ? it saves some time cause i always forget how it's called ^^

Reply:
yes fine